modules/system/assets/js/lang/lang.ru.js

```diff
@@ -6,7 +6,7 @@ if ($.oc === undefined) $.oc = $.wn
 );
 
 //! moment.js locale configuration v2.22.2
```

Comment:
Please revert the changes to the permissions on these two compiled files, they should be set to 644 by default in the repository, looks like when you compiled them it changed both compiled JS lang files to 755.

Reply:
Can you please recheck this? Because at my local environment they have **644** permission, just for curiosity I changed permissions to 755 and git showed new changes, so the original commits had 644 permission (well, at least at my local git repository).

Reply:
@arvislacis GitHub is definitely showing them as having been changed to 0755 permissions in your branch:

<img width="1259" alt="Screen Shot 2021-11-29 at 4 42 11 pm" src="https://user-images.githubusercontent.com/15900351/143835115-b2327358-b812-40da-a8f0-8448ce13b024.png">

Reply:
@LukeTowers @bennothommo Should be fixed in the latest commit. Thanks for noticing it. :)